src/ray/core_worker/object_interface.h

```diff
@@ -54,6 +55,12 @@ class CoreWorkerObjectInterface {
  private:
   /// Reference to the parent CoreWorker instance.
   CoreWorker &core_worker_;
+
+  /// Plasma store client.
+  plasma::PlasmaClient store_client_;
+
+  /// Default timeout of a get.
+  static const int c_get_timeout_ms_;
```

Comment:
we can use `RayConfig.get_timeout_milliseconds` as the default value.

Reply:
yep, thanks